Bottom: 75a3cf640a577cfecafec36a1379192f2fc6d61d
Top:    621d9e2cd461701dafe8d2d26424d4a78c1faf9f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-13 16:45:58 -0400

Add SNDRV_PCM_FMTBIT_S32_BE as a valid AC97 format


---

diff --git a/include/sound/soc-dai.h b/include/sound/soc-dai.h
index 496dc30..352d7ee 100644
--- a/include/sound/soc-dai.h
+++ b/include/sound/soc-dai.h
@@ -79,7 +79,8 @@ struct snd_pcm_substream;
 #define SND_SOC_CLOCK_OUT		1
 
 #define SND_SOC_STD_AC97_FMTS (SNDRV_PCM_FMTBIT_S16_LE |\
-                               SNDRV_PCM_FMTBIT_S32_LE)
+                               SNDRV_PCM_FMTBIT_S32_LE |\
+                               SNDRV_PCM_FMTBIT_S32_BE)
 
 struct snd_soc_dai_ops;
 struct snd_soc_dai;
